docs/specs/forecast-api.md

```diff
@@ -277,7 +260,15 @@ format:
   "features": [
       {
         "type": "Feature",
-        "properties": {},
+        "properties": {
+            "aircraft_class": "default",
+            "timestamp": "<%Y-%m-%dT%H:%M:%S>",
+            "flight_level": <int>,
+            "threshold": <int>,
+            "model_forecast_hour": <int>,
```

Comment:
Are we including any `model_...` properties anymore?

My suggestion would be to just include the `forecast_reference_time` only to be consistent with the netCDF file.  If you want to include `forecast_period` (instead of `model_forecast_hour`) this would be the integer hours between `timestamp` and `forecast_reference_time`

Or if you want to include these, then copy the definitions from up above down here. I would only include

Rather just the `forecast_reference_time` the same as in the netCDF file?

Reply:
I agree.

I think we can strike the `model_*` quantities (for two reasons).

1. nice to make this nomenclature an internal concern only
2. anybody concerning themselves with model run at vs. predicted at... will be well enough informed to know that the `"timestamp"` quantity here is equivalent to `"predicted_at"`, `"forecast_reference_time"` is equiv. to `"model_run_at"`, and `"model_forecast_hour"` is the diff of the two.